docs/001_develop/03_client-capabilities/008_interaction/checkbox.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Checkbox'
+sidebar_label: 'Checkbox'
+id: client-interaction-checkbox
+keywords: [interaction, interactive, checkbox, box, boolean, form, click, change, checked, toggle, submit, data]
+tags:
+- interaction
+- interactive
+- checkbox
+- box
+- boolean
+- form
+- click
+- change
+- checked
+- toggle
+- submit
+- data
+sidebar_position: 8
+---
+
+import CheckboxDemo from '../../../../examples/ui/client-capabilities/interaction/checkbox.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Checkbox
+
+A form-connected implementation of a [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) input.
+
+## Example
+
+<CheckboxDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
```

Comment:
I think we're looking good for the most part. I know you're not finished with PR so let me know if I'm pointing out something that will be addressed later.
- The checkbox demo I think it might be worth showing after the code example . I think it work better as a workflow. I found myself looking for it and then realized it was at the top. It might be missed
- Would it be worth having a summary or bullet point list of the configurations available for checkbox before the DOM API?
Lastly this is a question: is the DOM API pointing to FUI right now?

Reply:
To add to these, just to provoke thought, are we missing anything here like we had before if I look at current checkbox page:

1. Example use cases (maybe its obvious?) : https://learn.genesis.global/docs/previous/web/web-components/form/checkbox/?_highlight=checkbox#use-cases
2. How to get the current user selection https://learn.genesis.global/docs/previous/web/web-components/form/checkbox/?_highlight=checkbox#get-the-user-input

Reply:
Thanks for the thoughtful questions here

1. I am not so sure about moving the example below the code, I was thinking the entire idea was to sell the idea of the component first so the user knows if that is the component they need before having to look at any code. I completely agree that it's easy to miss though - I have altered the example to have some base react components that styling the example similarly to the code sections and hepefully that allows it to stand out easily on the page
2. What configurations are you referring to here? I would think any configurations are part of the DOM API section?
3. I have added the use cases section from the old page to this new file 8fe7a754a5d5afd4a4fd1043edf5a6ce33e8da69. They are quite obvious for this component, but I think it sets a good president for explaining the use case of more complicated components
4. That section is just showing you how to wire the checkbox to the property on the class, which is what the three code examples show that I have already added on the page.

Reply:
For your second point: What configurations are you referring to here? I would think any configurations are part of the DOM API section?
I understand they would be part of the DOM API but thought it might be helpful to have summary of the main ones as it could significantly enhance readability and user comprehension

Reply:
Ok I see what you mean, I am in two minds about this. I think for more complicated components it could be useful, but I feel that we already implicitly call out the main usages via the code examples here. To add additional configurations I feel would start to unnecessarily increase the length of the documentaiton and add a lot of repetition. Happy to be outvoted but I would lean towards not doing that for this type of the component (but the code examples should call out the main usages).